requirements.txt

```diff
@@ -0,0 +1,12 @@
+aspy.yaml==0.2.2
```

Comment:
I'm not sure,
I did it from virt environment, so I thought all from there are important